docs/matchers.md

```diff
@@ -204,6 +204,28 @@ it("passes if $closure echoes the expected output", function() {
 });
 ```
 
+**toMatchEcho($expected)**
```

Comment:
can update the `kahlan/docs` also : https://github.com/kahlan/docs/blob/master/matchers.md ?